paper/src/main/java/com/denizenscript/denizen/paper/properties/PaperPlayerExtensions.java

```diff
@@ -136,6 +149,32 @@ public static void register() {
             }
         });
 
+        // <--[mechanism]
+        // @object PlayerTag
+        // @name view_distance
+        // @input ElementTag(Number)
+        // @Plugin Paper
+        // @group paper
+        // @description
+        // Sets this player's view distance. Input must be a number between 2 and 32.
+        // This will be reset when a player rejoins. Provide empty input to unset.

```

Comment:
When allowing no input, it's usually no input at all (I.e. use the register method without a required param, and `Provide no input` in the meta).